### PR TITLE
Add max_length to fields in cost models

### PR DESCRIPTION
**Pull request by boaz0**
This is to sync with the UI input verification.

COST 781

//cc @dccurtis @dlabrecq @adberglund @kholdaway

### CODE REVIEW

File: koku/cost_models/serializers.py [hunk 1]
```diff
@@ -384,9 +384,9 @@ class Meta:
 
     uuid = serializers.UUIDField(read_only=True)
 
-    name = serializers.CharField(allow_blank=True)
+    name = serializers.CharField(max_length=100)
```
Comment (dccurtis) [inline koku/cost_models/serializers.py:387]: accidentally dropped `allow_blank`?

File: openshift/buildenv/flower/Pipfile [hunk 2]
```diff
@@ -4,7 +4,7 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-flower = "==0.9.2"
+flower = "==0.9.5"
```
Comment (dccurtis) [inline openshift/buildenv/flower/Pipfile:7]: You can revert the flower changes.  We're going to just remove flower from our production rollout in app interface.  Thanks!

**Comment by boaz0** [comment]
@dccurtis  Thanks for the review. I roll back some changes as you suggested. :space_invader:

**Review by dccurtis — APPROVED** [review]

**Comment by boaz0** [comment]
Thanks @dccurtis

**Pull request merged**